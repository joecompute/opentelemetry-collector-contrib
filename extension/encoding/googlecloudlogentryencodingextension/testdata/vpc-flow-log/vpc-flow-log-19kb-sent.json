{
  "insertId": "4qnfbvepj6ec",
  "jsonPayload": {
    "src_vpc": {
      "subnetwork_name": "default",
      "project_id": "test-project-id",
      "subnetwork_region": "asia-south1",
      "vpc_name": "default"
    },
    "reporter": "SRC",
    "dest_location": {
      "continent": "Asia",
      "asn": 137718,
      "country": "chn"
    },
    "bytes_sent": "19680",
    "start_time": "2025-09-27T21:12:57.083366649Z",
    "src_instance": {
      "vm_name": "service-integration-dev-idc-ubuntu25-4",
      "project_id": "test-project-id",
      "region": "asia-south1",
      "zone": "asia-south1-c"
    },
    "connection": {
      "dest_port": 52388,
      "src_ip": "10.160.0.36",
      "src_port": 22,
      "dest_ip": "101.126.151.131",
      "protocol": 6
    },
    "end_time": "2025-09-27T21:12:58.512821056Z",
    "internet_routing_details": {
      "egress_as_path": [
        {
          "as_details": [
            {
              "asn": 58453
            },
            {
              "asn": 9808
            },
            {
              "asn": 38019
            },
            {
              "asn": 137718
            }
          ]
        }
      ]
    },
    "packets_sent": "60"
  },
  "resource": {
    "type": "vpc_flow_logs_config",
    "labels": {
      "name": "joe-test-flow-logs-default",
      "location": "global",
      "resource_container": "projects/123456789012"
    }
  },
  "timestamp": "2025-09-27T21:13:04.502203447Z",
  "labels": {
    "target_resource_name": "projects/123456789012/global/networks/default"
  },
  "logName": "projects/test-project-id/logs/networkmanagement.googleapis.com%2Fvpc_flows",
  "receiveTimestamp": "2025-09-27T21:13:04.502203447Z"
}