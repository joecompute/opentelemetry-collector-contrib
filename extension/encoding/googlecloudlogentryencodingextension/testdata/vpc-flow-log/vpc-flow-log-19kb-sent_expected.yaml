resourceLogs:
  - resource:
      attributes:
        - key: gcp.project
          value:
            stringValue: elastic-obs-integrations-dev
        - key: cloud.resource_id
          value:
            stringValue: networkmanagement.googleapis.com%2Fvpc_flows
        - key: gcp.resource_type
          value:
            stringValue: vpc_flow_logs_config
        - key: gcp.label.name
          value:
            stringValue: "joe-test-flow-logs-default"
        - key: gcp.label.location
          value:
            stringValue: global
        - key: gcp.label.resource_container
          value:
            stringValue: projects/774712120909
    scopeLogs:
      - logRecords:
          - attributes:
              - key: log.record.uid
                value:
                  stringValue: 4qnfbvepj6ec
              - key: gcp.label.target_resource_name
                value:
                  stringValue: projects/774712120909/global/networks/default
              - key: network.transport
                value:
                  stringValue: tcp
              - key: source.address
                value:
                  stringValue: 10.160.0.36
              - key: destination.address
                value:
                  stringValue: 101.126.151.131
              - key: source.port
                value:
                  intValue: "22"
              - key: destination.port
                value:
                  intValue: "52388"
              - key: gcp.vpc.flow.reporter
                value:
                  stringValue: SRC
              - key: gcp.vpc.flow.bytes_sent
                value:
                  intValue: "19680"
              - key: gcp.vpc.flow.packets_sent
                value:
                  intValue: "60"
              - key: gcp.vpc.flow.start_time
                value:
                  stringValue: "2025-09-27T21:12:57.083366649Z"
              - key: gcp.vpc.flow.end_time
                value:
                  stringValue: "2025-09-27T21:12:58.512821056Z"
              - key: gcp.vpc.flow.source.instance.project.id
                value:
                  stringValue: elastic-obs-integrations-dev
              - key: gcp.vpc.flow.source.instance.vm.region
                value:
                  stringValue: asia-south1
              - key: gcp.vpc.flow.source.instance.vm.name
                value:
                  stringValue: service-integration-dev-idc-ubuntu25-4
              - key: gcp.vpc.flow.source.instance.vm.zone
                value:
                  stringValue: asia-south1-c
              - key: gcp.vpc.flow.source.project.id
                value:
                  stringValue: elastic-obs-integrations-dev
              - key: gcp.vpc.flow.source.subnet.name
                value:
                  stringValue: default
              - key: gcp.vpc.flow.source.subnet.region
                value:
                  stringValue: asia-south1
              - key: gcp.vpc.flow.source.vpc.name
                value:
                  stringValue: default
              - key: gcp.vpc.flow.destination.asn
                value:
                  intValue: "137718"
              - key: gcp.vpc.flow.destination.geo.continent
                value:
                  stringValue: Asia
              - key: gcp.vpc.flow.destination.geo.country.iso_code.alpha3
                value:
                  stringValue: chn
              - key: gcp.vpc.flow.egress.as_paths
                value:
                  arrayValue:
                    values:
                      - kvlistValue:
                          values:
                            - key: as_details
                              value:
                                arrayValue:
                                  values:
                                    - kvlistValue:
                                        values:
                                          - key: asn
                                            value:
                                              intValue: "58453"
                                    - kvlistValue:
                                        values:
                                          - key: asn
                                            value:
                                              intValue: "9808"
                                    - kvlistValue:
                                        values:
                                          - key: asn
                                            value:
                                              intValue: "38019"
                                    - kvlistValue:
                                        values:
                                          - key: asn
                                            value:
                                              intValue: "137718"
            body: {}
            observedTimeUnixNano: "1759007584502203447"
            timeUnixNano: "1759007584502203447"
        scope: {}
